control/mpc_lateral_controller/test/test_mpc.cpp

```diff
@@ -168,7 +168,8 @@ class MPCTest : public ::testing::Test
     mpc.m_admissible_position_error = admissible_position_error;
     mpc.m_admissible_yaw_error_rad = admissible_yaw_error_rad;
     mpc.m_steer_lim = steer_lim;
-    mpc.m_steer_rate_lim = steer_rate_lim;
+    mpc.m_steer_rate_lim_map_by_curvature.push_back({0.0, steer_rate_lim});
+    mpc.m_steer_rate_lim_map_by_velocity.push_back({0.0, steer_rate_lim});
```

Comment:
mino thoughr: use `emplace_back()` when you call the default constructor.

```suggestion
    mpc.m_steer_rate_lim_map_by_velocity.emplace_back(0.0, steer_rate_lim);
```

Reply:
DONE